src/Runner.js

```diff
@@ -154,7 +154,7 @@ function run(transformFile, paths, options) {
           endTime[0],
           (endTime[1]/1000000).toFixed(0)
         );
-        resolve();
```

Comment:
`resolve` is not colleable, it is just the array with the results of `pendingWorkers` promises.

resolving to `fileCounters` seems more useful when runner is called programmatically instead of via CLI.